mergify_engine/engine/commands_runner.py

```diff
@@ -83,14 +86,23 @@ class CommandInvalid(Exception):
     message: str
 
 
+@dataclasses.dataclass
+class NotACommand(Exception):
+    message: str
+
+
 def load_command(
     mergify_config: rules.MergifyConfig,
     message: str,
 ) -> Command:
     """Load an action from a message."""
     action_classes = actions.get_commands()
     match = COMMAND_MATCHER.search(message)
-    if match and match[1] in action_classes:
+
+    if not match:
+        raise NotACommand(COMMENT_IS_NOT_A_COMMAND)
```

Comment:
You can probably just embed the str here, no need to have a variable IMHO. We're not going to reuse it and it makes the code a little bit harder to follow.